pkg/env/dev.go

```diff
@@ -165,9 +165,13 @@ func (d *dev) InitializeAuthorizers() error {
 
 func (d *dev) AROOperatorImage() string {
 	override := os.Getenv("ARO_IMAGE")
+	tag := version.GitCommit
 	if override != "" {
 		return override
 	}
+	if tag == "unknown" {
+		tag = "latest"
+	}
 
 	return fmt.Sprintf("%s.azurecr.io/aro:%s", d.acrName, version.GitCommit)
```

Comment:
```suggestion
	return fmt.Sprintf("%s.azurecr.io/aro:%s", d.acrName, tag)
```
saying that I am not convinced we need this, GitCommit is not used in the CI case..

Reply:
It is used in local run. If you are running from check-out replica it is trying to pull that commit image. But once you change any file in the repo but don't commit - it gets unknown.

Reply:
ok, but that is as a developer and you should just be setting ARO_IMAGE=
can you remove this and let's merge the first chuck and chat about this after..
I had this earlier and Jim didn't like it and said that devs should just always set ARO_IMAGE

Reply:
If developers should always set it, let's add it default and people can override. We already have enough information for people to sink it. Because now behaviour is questionable. Previously - default settings - all works, now - not so much.

Reply:
https://github.com/Azure/ARO-RP/issues/925 to discuss